react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -115,6 +175,10 @@ class ParticipantView extends Component {
         // FIXME It's currently impossible to have true as the value of
         // waitForVideoStarted because videoTrack's state videoStarted will be
         // updated only after videoTrack is rendered.
+        // XXX Note that, unlike on web, we don't render video when the
```

Comment:
But since we control the renderer it could be done (...eventually - if we would really, really want to have that) ?

Reply:
Potentially yes, but we only control the rendering to a degree. I was just documenting the current state of affairs here :-)